RetrospectiveExtension.Frontend/components/feedbackCarousel.tsx

```diff
@@ -68,10 +75,38 @@ class FeedbackCarousel extends React.Component<IFeedbackCarouselProps, IFeedback
         {this.props.feedbackColumnPropsList.map((columnProps) => {
           const mainCardCount = columnProps.columnItems.filter((columnItem) => !columnItem.feedbackItem.parentFeedbackItemId).length;
 
+          columnProps.columnItems.forEach(columnItem => {
+            const feedbackItemProps = FeedbackColumn.createFeedbackItemProps(columnProps, columnItem, true);
+
+            // Establish whether an item in the column has children feedback grouped beneath it,
+            // and therefore will be the parent
+            const isGroupedCarouselItem = columnItem.feedbackItem.childFeedbackItemIds ? (columnItem.feedbackItem.childFeedbackItemIds.length > 0 ? true : false) : false;
+
+            // Set that property for later
+            columnItem.feedbackItem.isGroupedCarouselItem = columnItem.feedbackItem.childFeedbackItemIds ? (columnItem.feedbackItem.childFeedbackItemIds.length > 0 ? true : false) : false;
+
+            if (isGroupedCarouselItem) {
+              // If an item in the column is a parent, get the title of the children
+              // to show in 'Related Feedback' section
+              const columnItemChildrenIds = columnItem.feedbackItem.childFeedbackItemIds;
+              const childrenTitlesShort: String[] = [];
+              columnItemChildrenIds.forEach(childId => {
+                // For every child item in the group, limit to 2 lines (~200 characters)
+                const childFeedbackItem = columnProps.columnItems.find(childItem => childItem.feedbackItem.id == childId)
+                const origTitle = childFeedbackItem.feedbackItem.title
+                const shortTitle = origTitle.length > 200 ? origTitle.substring(0, 200) + '...' : origTitle;
```

Comment:
nit: I think `200` should be a const at the beginning of the file, making it easier to change this value in the future if the limit for child length changes.

Reply:
Great idea!